hydrolib/core/dflowfm/extold/models.py

```diff
@@ -0,0 +1,417 @@
+    """Union[PolyFile, DiskOnlyFileModel]: The file associated to this forcing."""
+
+    varname: Optional[str] = Field(None, alias="VARNAME")
+    """Optional[str]: The variable name used in `filename` associated with this forcing; some input files may contain multiple variables."""
+
+    sourcemask: DiskOnlyFileModel = Field(
+        default_factory=lambda: DiskOnlyFileModel(None), alias="SOURCEMASK"
+    )
+    """DiskOnlyFileModel: The file containing a mask."""
+
+    filetype: FileType = Field(alias="FILETYPE")
+    """FileType: Indication of the file type.
+    
+    Options:
+    1. Time series
+    2. Time series magnitude and direction
+    3. Spatially varying weather
+    4. ArcInfo
+    5. Spiderweb data (cyclones)
+    6. Curvilinear data
+    7. Samples (C.3)
+    8. Triangulation magnitude and direction
+    9. Polyline (<*.pli>-file, C.2)
+    11. NetCDF grid data (e.g. meteo fields)
+    14. NetCDF wave data
+    """
+
+    method: Method = Field(alias="METHOD")
+    """Method: The method of interpolation.
+    
+    Options:
+    1. Pass through (no interpolation)
+    2. Interpolate time and space
+    3. Interpolate time and space, save weights
+    4. Interpolate space
+    5. Interpolate time
+    7. Interpolate/Extrapolate time
+    """
+
+    operand: Operand = Field(alias="OPERAND")
+    """Operand: Overwriting or superimposing values already set for this quantity:
+    'O' Values are overwritten.
+    '+' New value is superimposed.
+    """
+
+    value: Optional[float] = Field(None, alias="VALUE")
+    """Optional[float]: custom coefficients for transformation."""
+
+    factor: Optional[float] = Field(None, alias="FACTOR")
+    """Optional[float]: The factor."""
+
+    ifrctyp: Optional[float] = Field(None, alias="IFRCTYP")
+    """Optional[float]: The friction type."""
+
+    averagingtype: Optional[float] = Field(None, alias="AVERAGINGTYPE")
+    """Optional[float]: The averging type."""
+
+    relativesearchcellsize: Optional[float] = Field(
+        None, alias="RELATIVESEARCHCELLSIZE"
+    )
+    """Optional[float]: The relative search cell size."""
+
+    extrapoltol: Optional[float] = Field(None, alias="EXTRAPOLTOL")
+    """Optional[float]: The extrapolation tolerance."""
+
+    percentileminmax: Optional[float] = Field(None, alias="PERCENTILEMINMAX")
+    """Optional[float]: The percentile min max."""
+
+    area: Optional[float] = Field(None, alias="AREA")
+    """Optional[float]: The area for sources and sinks."""
+
+    nummin: Optional[int] = Field(None, alias="NUMMIN")
+    """Optional[int]: The nummin."""
+
+    @validator("quantity", pre=True)
+    def validate_quantity(cls, value):
+        if isinstance(value, str):
+            lower_value = value.lower()
+
+            if lower_value.startswith(Quantity.TracerBnd):
+                n = len(Quantity.TracerBnd.value)
+                return Quantity.TracerBnd.value + value[n:]
```

Comment:
Do we also have to check that we atleast have _something_ for the name part of the tracer? Or is the following also valid:

`model.quantity = Quantity.TracerBnd`

(which will result in the quantity only being 'tracerbnd'). Or is that not hydrolib-core's responsibility?